apps/condo/domains/ticket/utils/serverSchema/TicketDocumentGeneration/paidWorks.js

```diff
@@ -0,0 +1,157 @@
+        })
+        : null
+
+    const invoices = await Invoice.getAll(context, {
+        ticket: { id: ticket.id },
+        deletedAt: null,
+        status_not: INVOICE_STATUS_CANCELED,
+    }, 'id currencyCode accountNumber rows { isMin toPay name count vatPercent } recipient { bankName bankAccount bic }', {
+        sortBy: ['createdAt_ASC'],
+    })
+
+    const currencyCode = get(invoices, '0.currencyCode') || DEFAULT_INVOICE_CURRENCY_CODE
+    let totalSum = Big(0), totalVAT = Big(0)
+
+    const listOfWorks = invoices.reduce((acc, invoice) => {
+        const rows = Array.isArray(invoice.rows) ? invoice.rows : []
+        acc.push(...rows.map((row, index) => {
+            const price = Big(!row.isMin ? row.toPay : 0)
+            const sum = price.times(get(row, 'count', 1))
+            const vatPercent = Big(get(row, 'vatPercent', 0))
+            totalSum = totalSum.plus(sum)
+            totalVAT = totalVAT.plus(sum.times(vatPercent).div(100))
+
+            return {
+                number: index + 1,
+                name: row.name || '',
+                count: String(row.count) || '',
+                price: !Number.isNaN(price) ? renderMoney(price, currencyCode, locale) : '',
+                vat: !Number.isNaN(vatPercent) ? renderMoney(vatPercent, currencyCode, locale) : '',
+                sum: !Number.isNaN(sum) ? renderMoney(sum, currencyCode, locale) : '',
+            }
+        }))
+
+        return acc
+    }, [])
+
+    totalSum.toFixed(2)
+    totalVAT.toFixed(2)
+
+    const documentData = {
+        header: {
+            generalDate: printDate.format('DD.MM.YYYY'),
+        },
+        company: {
+            name: get(organization, 'name') || buildEmptyLineDifferentLength(true),
+            psrn: psrn || buildEmptyLineDifferentLength(),
+            tin: get(organization, 'tin') || buildEmptyLineDifferentLength(),
+            iec: iec || buildEmptyLineDifferentLength(),
+            address: organizationAddress || buildEmptyLineDifferentLength(true),
+            phone: get(organization, 'phone') || buildEmptyLineDifferentLength(),
+
+        },
+        bankDetails: {
+            accountNumber: get(invoices, '0.accountNumber') || buildEmptyLineDifferentLength(),
+            bankName: get(invoices, '0.recipient.bankName') || buildEmptyLineDifferentLength(),
+            bankAccount: get(invoices, '0.recipient.bankAccount') || buildEmptyLineDifferentLength(),
+            bic: get(invoices, '0.recipient.bic') || buildEmptyLineDifferentLength(),
+        },
+        client: {
+            name: get(contact, 'name') || '',
+        },
+        listOfWorks,
+        sum: {
+            totalSum: !Number.isNaN(totalSum) ? renderMoney(totalSum, currencyCode, locale) : '',
+            totalVAT: !Number.isNaN(totalVAT) ? renderMoney(totalVAT, currencyCode, locale) : '',
+            totalSumInWords: buildEmptyLineDifferentLength(true),
+            totalVATInWords: buildEmptyLineDifferentLength(true),
```

Comment:
It is difficult for us to translate numbers into words
- we don't want to drag some lib (for various reasons)
- we don't want to make our own lib (it is difficult/long/needs support)
- we don't want to make crutches either :)